.github/workflows/ci.yml

```diff
@@ -229,7 +244,7 @@ jobs:
           app_hostname: ${{ inputs.app_hostname }}
           branch: ${{ inputs.branch }}
           deploy_root: app/${{ inputs.deploy_root }}
-          deploy_labels: gh/workflow-version=${{ inputs.ci_version }}
+          deploy_labels: gh/workflow-version=${{ needs.build.outputs.ci_version }}
```

Comment:
Needs to be `needs.build.outputs.workflow_ci_version` (Hint: See how `image_ref ` output is defined in `build`)